src/atoms/exp_cone/LogSumExpAtom.jl

```diff
@@ -32,7 +32,7 @@ function new_conic_form!(context::Context, e::LogSumExpAtom)
     # log(sum(exp(x))) <= t  <=>  sum(exp(x)) <= exp(t) <=> sum(exp(x - t)) <= 1
     t = Variable()
     z = sum(exp(e.children[1] - t * ones(size(e.children[1]))))
-    add_constraint!(context, z <= 1)
+    add_constraint!(context, 1 >= z)
```

Comment:
Changing these just means we don't have to update the tests, and they use the more popular `Nonnegatives` instead of `Nonpositives`.